cpp/include/Ice/Communicator.h

```diff
@@ -323,12 +323,12 @@ namespace Ice
         [[nodiscard]] IceInternal::ReferencePtr _propertyToProxy(std::string_view property) const;
 
         /// @cond INTERNAL
-        friend ICE_API CommunicatorPtr initialize(int&, const char*[], InitializationData);
-        friend ICE_API CommunicatorPtr initialize(InitializationData);
+        friend ICE_API_FRIEND CommunicatorPtr initialize(int&, const char*[], InitializationData);
```

Comment:
Required on Windows, but doesn't work with the standard attribute placement (as the end).